### DISCUSSION TITLE
我做程序员这十年

### DISCUSSION THREAD

**Discussion by giscus** [keelii/fresh-blog#3] [Announcements]
# 我做程序员这十年

芝兰生于深林，不以无人而不芳，君子修道立德，不为穷困而败节。

https://keelii.com/2022/05/19/ten-years-programming

**Comment by DKSCIcat** [comment]
大佬现在在哪工作呢

**Reply by keelii** [reply]
JD.com

**Reply by cghbh** [reply]
厉害

**Comment by ZhangAliya** [comment]
大佬  你可以谈谈现在的就业形势么？在学校里总是听见一些传闻，但是并不清楚真实的情况。

**Reply by keelii** [reply]
好久没换工作了，谈就业形势我觉得我没这个水平。

**Comment by MarvinXu** [comment]
能干10年不被裁也需要本事

**Comment by qualvey** [comment]
催更
